cypress/e2e/signIn.cy.js

```diff
@@ -28,7 +28,31 @@ describe('Sign In page', () => {
     homePage.assertHeaderContainUsername(user.username);
   });
 
-  it('should not provide an ability to log in with wrong credentials', () => {
+  it('should not provide an ability to log in with wrong email', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    const userEmail = '77' + user.email;
+    signInPage.typeEmail(userEmail);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
+
+    cy.contains('Login failed!');
+    cy.contains('Invalid user credentials');
+    cy.contains('OK').click();
+  });
+
+  it('should not provide an ability to log in with wrong password', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    signInPage.typeEmail(user.email);
+    const userPassword = '77' + user.password;
+    signInPage.typePassword(userPassword);
+    signInPage.clickSignInBtn();
 
+    cy.contains('Login failed!');
```

Comment:
Similar to the above, consider using data-qa attributes for selecting elements.

Reply:
this element doesn't have data-qa attributes
https://mate-parttime.slack.com/archives/C02J1F3EUDB/p1691237811581409